include/eve/detail/top_bits.hpp

```diff
@@ -55,7 +55,7 @@ EVE_FORCEINLINE auto movemask_raw(Logical p)
   using ABI = abi_type_t<Logical>;
   using e_t = element_type_t<Logical>;
 
-       if constexpr( !ABI::is_wide_logical ) return p.storage();
```

Comment:
This is definetly ius wide logical. Really plz don't touch this one - I'm making changes here